VetClinic.WebApi/ViewModels/OrderProcedureViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace VetClinic.WebApi.ViewModels
+{
+    public class OrderProcedureViewModel
+    {
+        public int Id { get; set; }
+        public int Count { get; set; }
+        public TimeSpan Time { get; set; }
+        public string Conclusion { get; set; }
+        public string Details { get; set; }
+        public int OrderId { get; set; }
+        public int AppointmentId { get; set; }
+        public int ProcedureId { get; set; }
+        public int PetId { get; set; }
+        public int EmployeeId { get; set; }
```

Comment:
these properties are good by receiving data from client side, but you dont provide any information to client side, such as employee that makes a procedure, or appointment date